packages/replayio/src/utils/recordings/formatRecording.ts

```diff
@@ -7,58 +8,57 @@ const MAX_TITLE_LENGTH = 35;
 export function formatRecording(recording: LocalRecording) {
   const id = recording.id.substring(0, 8) + "…";
 
+  const { runtime } = parseBuildId(recording.buildId);
+
   let title;
-  if (recording.metadata.host) {
-    if (recording.metadata.host.length > MAX_TITLE_LENGTH) {
-      title = link(recording.metadata.host.substring(0, MAX_TITLE_LENGTH).trimEnd() + "…");
-    } else {
-      title = link(recording.metadata.host);
+  switch (runtime) {
+    case "node": {
+      title = "NodeJS";
```

Comment:
Small proposal to make this one more robust 
https://github.com/replayio/replay-cli/pull/354